types/d3-scale/index.d.ts

```diff
@@ -714,8 +779,10 @@ export function scaleSymlog<Range, Output = Range>(
 /**
  * Identity scales are a special case of linear scales where the domain and range are identical; the scale and its invert method are thus the identity function.
  * These scales are occasionally useful when working with pixel coordinates, say in conjunction with an axis.
+ *
+ * The generic corresponds to the data type of the unknown value.
  */
-export interface ScaleIdentity {
+export interface ScaleIdentity<Unknown> {
```

Comment:
I think you want to default this to `never`. Otherwise it defaults to `unknown` (lowercase `u`, the TS builtin) and `number | unknown` is `unknown`, which will make this interface unusable.

Reply:
I don't think it will default to `never`, it will just complain that you didn't provide a generic.

Reply:
Yes, you're right about `unknown`. I was thinking of cases where TS can't infer a type.

But in this case adding a default of `never` would avoid breaking changes for anyone currently using this type. What's the downside?

Reply:
It can make it easier to forget to provide it in places that it should be specified. Either way, I went ahead and made the change.